performance/gradle/benchmark.gradle

```diff
@@ -96,7 +96,7 @@ task konanJsonReport {
                                                   'compilerVersion': "${konanVersion}".toString(),
                                                   'flags': kotlin.targets.native.compilations.main.extraOpts.collect{ "\"$it\"" },
                                                   'benchmarks': benchContents,
-                                                  'compileTime': nativeCompileTime,
+                                                  'compileTime': [nativeCompileTime],
```

Comment:
Why list? What else shall be here?

Reply:
There can be several measurements of compile time to get more stable result.